src/main/java/io/neonbee/config/metrics/MicrometerRegistryLoader.java

```diff
@@ -0,0 +1,20 @@
+package io.neonbee.config.metrics;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.vertx.micrometer.MicrometerMetricsOptions;
+
+/**
+ * Interface to implement if you want to add a {@link MeterRegistry} to the {@link MicrometerMetricsOptions}.
+ * <p>
+ * The implementing class MUST have a constructor without any arguments.
+ */
+@FunctionalInterface
+public interface MicrometerRegistryLoader {
```

Comment:
Why is this class required?

Reply:
We cannot load the MeterRegistry directly because sometimes it needs to be configured. For example, the DynatraceMeterRegistry requires some configuration. Of course, we could provide the Micrometer configuration map, but then we have to deal with the next problem that the MeterRegistry implementations do not all have the same constructor signature.